### PR TITLE
Dependency tracking on Azure DevOps 09/07

### PR DESCRIPTION
**Pull request by wpschaub**
Summarizing a working group's finding of dependency tracking.

### CODE REVIEW

**Review by AndreKaminski — COMMENTED** [review]
Love the analogy!

**Review by danielrbroderick — APPROVED** [review]

**Comment by wsbctechnicalblog** [comment]
Merging, based on another inspiring confirmation from @sben101  - "well written! The analogy and explanation makes it easy to follow and read."

File: content/azure-devops-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+3. Dependency work item (489411) for Merlin engine is on the backlog, linked as a child to Booster Feature (489408).
+4. Merlin engine Story Enabler (489419) is still on the team 2 backlog, which implies that the common services Engine team is not able to commit to the work item yet.
+
+> ![Board 2](../images/azure-devops-dependencies-7.png) 
+
+Switching to the **Feature** board view, we have a visual cue on both the 2-stage and Booster cards that they have dependencies. Users can click on the dependencies to get more detail and to follow the dependency links.
+
+> ![Dependency Extension](../images/azure-devops-dependencies-10.png) 
+
+The team also uses the optional [Dependency Tracker](https://marketplace.visualstudio.com/items?itemName=ms-eswm.dependencytracker) extension to get visual information on their dependencies, timelines, and risks. The extension is not for the faint hearted but delivers phenomenal dependency details when used currently.
```
Comment (MerlinDarkstar) [inline content/azure-devops-dependencies.md:84]: "...delivers phenomenal dependency details when used currently." ... should that be ".... used correctly"?

Comment (wpschaub) [reply content/azure-devops-dependencies.md:84]: @MerlinDarkstar looking at the master and the live blog, this was corrected a while ago:
image

**Pull request merged**